.travis.yml

```diff
@@ -1,4 +1,7 @@
+sudo: false
 language: python
+python:
+  - "3.5"
```

Comment:
no, tox is still the one responsible to run tests in each version.
This change instruct travis-ci to install locally python3.5 which is not provided by default.